lib/AppInfo/Migrations/Version0001Date20211201101630.php

```diff
@@ -0,0 +1,100 @@
+<?php
+
+namespace OCA\ScienceMesh\Migration;
+
+use Closure;
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\DBAL\Types\Types;
+use OCP\IDBConnection;
+use OCP\Migration\IOutput;
+use OCP\Migration\ISchemaMigration;
+
+
+class Version0001Date20211201101630 implements ISchemaMigration {
+
+	/** @var IDBConnection */
+
+	public function __construct(IDBConnection $connection) {
+		$this->connection = $connection;
+	}
+
+	public function changeSchema(Schema $schema, array $options) {
+
+        $prefix = $options['tablePrefix'];
+
+        $shares = $schema->createTable("{$prefix}sciencemesh_shares");
```

Comment:
Oh wow, this introduces a new database table. Is this for incoming or for outgoing shares?
This is a big change, let's have a meeting about this first and take a decision about it first! I'll ping you and @parhamin2010 on Slack.

Reply:
I cannot remember that I created this schema, I can remember there was an migration inside nc-sciencemesh and this schema is just a copy of that.